lib/rules/accessor-pairs.js

```diff
@@ -63,3 +63,11 @@ module.exports = function(context) {
     };
 
 };
+
+module.exports.documentation = {
+    errors: ["Getter is not present", "Setter is not present"],
+    description: "enforces getter/setter pairs in objects",
+    ruleType: "Best Practices",
+    recommended: false,
+    default: [{getWithoutSet: false, setWithoutGet: true}]
```

Comment:
That alone doesn't solve the synchronization issue since rules sometimes do their own processing to merge options with defaults. Would using [merge](https://www.npmjs.com/package/merge) with `options = merge.recursive(true, schema.default, context.options)` do the trick?